vmdb/db/migrate/20100908214807_recreate_relationships.rb

```diff
@@ -19,7 +19,8 @@ def self.up
     remove_index :relationships, :relationship
     remove_index :relationships, [:parent_type, :parent_id]
     remove_index :relationships, [:child_type, :child_id]
-    drop_table :relationships, :relationships_orig
+    drop_table :relationships
+    drop_table :relationships_orig if table_exists?(:relationships_org)
```

Comment:
I don't think so?  When I ran the migration, there was no `relationships_orig` table, so I threw on a `table_exists?` in case there was one in some environment.  Should we just remove that line?